hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -2447,6 +2447,11 @@ public Builder withAllowMultiWriteOnSameInstant(boolean allow) {
       return this;
     }
 
+    public Builder withArchiveBeyondSavepoint(boolean allow) {
```

Comment:
Would be preferable to have single builder method for this config. Can't we reuse the one in `HoodieArchivalConfig`?